ui/src/pages/backoffice/Patron/PatronDetails/components/ItemsSearch/components/ItemsResultsList/ItemsResultsList.js

```diff
@@ -118,14 +137,30 @@ export class ItemsResultsList extends Component {
       },
     ];
 
-    return results.hits.length ? (
-      <ResultsTable
-        data={results.hits}
-        columns={columns}
-        totalHitsCount={results.length}
-        name={'items'}
-      />
-    ) : null;
+    return results.hits && results.hits.length ? (
+      <Loader isLoading={checkoutIsLoading}>
+        <p>Found {results.hits.length} item(s).</p>
+        <ResultsTable
+          data={results.hits}
+          columns={columns}
+          totalHitsCount={results.length}
+          name={'items'}
+        />
+      </Loader>
+    ) : (
+      <Segment placeholder textAlign="center">
+        <Header icon>
+          <Icon name="search" />
+          Found no items matching this barcode.
+        </Header>
+        <p>HINT: Check if the book is not already on loan by someone else.</p>
+        <Segment.Inline>
+          <Button primary onClick={() => this.props.clearSearchQuery()}>
```

Comment:
```suggestion
          <Button primary onClick={this.props.clearSearchQuery}>
```

Reply:
same story, it will be called with the event as a param